packages/fxa-auth-server/docs/swagger/auth-server-api.ts

```diff
@@ -291,7 +291,7 @@ export const AUTH_SERVER_API_DESCRIPTION = {
             "errno": 201,
             "error": "Service Unavailable",
             "message": "Service unavailable",
-            "info": "https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/api.md#response-format",
+            "info": "https://mozilla.github.io/ecosystem-platform/api#section/Response-format",
```

Comment:
Could this be worth adding to or creating a `const` somewhere near our API docs so if this ever changes we can update it in one spot?

Reply:
In this case it changes so rarely (maybe never!) and it's an easily grepped string I don't think we need to maintain a variable.  Let's land it.